app/jobs/identities_backfill_job.rb

```diff
@@ -0,0 +1,54 @@
+class IdentitiesBackfillJob < ApplicationJob
+  # This is a short-term solution to backfill data requiring a table scan.
+  # This job can be deleted once it's done.
+
+  queue_as :low
+
+  # Let's give us the option to fine-tune this on the fly
+  BATCH_SIZE_KEY = 'IdentitiesBackfillJob.batch_size'.freeze
+  SLICE_SIZE_KEY = 'IdentitiesBackfillJob.slice_size'.freeze
+  CACHE_KEY = 'IdentitiesBackfillJob.position'.freeze
+
+  def perform
+    start_time = Time.now
+    max_id = ServiceProviderIdentity.last.id
+
+    (batch_size / slice_size).times.each do |slice_num|
+      start_id = position + (slice_size * slice_num)
+      next if start_id > max_id
+      sp_query = <<~SQL
+UPDATE identities
+SET last_consented_at = created_at
+WHERE id > #{start_id}
+AND id <= #{start_id + slice_size}
```

Comment:
question: did the query planner do smarter things with a `<= max` vs a `LIMIT` ?

Reply:
For reasons that escape me, I got a syntax error when I was doing `LIMIT`. I assumed the error was actually somewhere else there, tried expressing it like this to highlight what the problem actually was, and the error went away.

I think something silly is wrong with my query and I'm just not seeing it. See my other comment down in the test; this query is populating `last_consented_at` for rows that have `deleted_at` set despite us specifying not to do that.

For background, I know MySQL much better than Postgres. The former isn't known for being a rigid adherent to ANSI standard SQL, so am I maybe doing something MySQL-specific and not realizing it?

Reply:
ah! I think that the `LIMIT` has to be in a subquery: https://dba.stackexchange.com/questions/69471/postgres-update-limit-1

Reply:
I left this as an id range. While it wouldn't have been too hard to use a subquery here, it just felt really weird.